src/pybind/mgr/telemetry/module.py

```diff
@@ -255,6 +282,8 @@ def compile_report(self, channels=[]):
                 'features': mon_map['features']
             }
 
+            report['config'] = self.gather_configs()
```

Comment:
might want to update qa/tasks/mgr/dashboard/test_mgr_module.py accordingly . see #29361.